vnext/PropertySheets/External/Microsoft.ReactNative.WindowsSdk.Default.props

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- 
+  Copyright (c) Microsoft Corporation.
+  Licensed under the MIT License.
+-->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <!--
+    This defines the default Windows SDK used to build both the app and 0.66+
+    template community modules.
+  -->
+  <PropertyGroup Label="Globals">
+    <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
```

Comment:
This undoes the change that Julio made to use 19041

Reply:
Good catch... Not sure where I copied this from